ha_cluster_exporter.go

```diff
@@ -94,6 +94,21 @@ func main() {
 		prometheus.MustRegister(drbdCollector)
 	}
 
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(`<html>
+			<head>
+				<title>HACluster Exporter</title>
+			</head>
+			<body>
+				<h1>HACluster Exporter</h1>
+				<p><a href="/metrics">Metrics</a></p>
```

Comment:
I would make this a relative link, we don't know if this is going to be deployed under a base path or not.